### PR TITLE
refactor(support): convert support panel to direct db access

### LINKED ISSUE #9724: Convert support-panel to fxa-shared/db
Instead of auth-db-mysql server

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-3684)

### PR DESCRIPTION
**Pull request by dannycoates**
fxa-auth-db-mysql is deprecated. this converts the support-panel to use fxa-shared and with direct db access.

closes #9724

config PR https://github.com/mozilla-services/cloudops-deployment/pull/4373

### CODE REVIEW

**Comment by dannycoates** [comment]
Let's wait until after 214 to merge

**Review by bbangert — APPROVED** [review]
LGTM! Will this require deploy config notes, or is that `make_auth_config` going to align exactly with existing credentials?

**Comment by dannycoates** [comment]
> Will this require deploy config notes, or is that `make_auth_config` going to align exactly with existing credentials?

the PR here should cover it https://github.com/mozilla-services/cloudops-deployment/pull/4373

**Comment by dannycoates** [comment]
I'll wait until @jbuck gets back before merging this so that we can coordinate the config / deployment changes

**Pull request merged**
